docs/quickstart/subscribe-dataset.md

```diff
@@ -1,27 +1,6 @@
----
-title: Send a request guide
-summary: Description of how to send a request for a Dataset.
-authors:
-    - Patrick Lamplmair
----
-
-# Request a dataset guide
-
-In this guide you will learn how-to send a Request via the admin web-interface of your DataSpace Node. This guide covers the following topics:
-
-- [Prerequisites](/guides/guide-request-dataset/#prerequisites)
-- [Data preview](/guides/guide-request-dataset/#data-preview)
-- [Configure a request](/guides/guide-request-dataset/#configure-a-request)
-    - [Select sources and streams](/guides/guide-request-dataset/#select-sources-and-streams)
-    - [Select a time-frame](/guides/guide-request-dataset/#select-a-time-frame)
-    - [Accept terms and conditions](/guides/guide-request-dataset/#accept-terms-and-conditions)
-    - [Send a request](/guides/guide-request-dataset/#send-a-request)
-- [The state of your Subscription](/guides/guide-request-dataset/#the-state-of-your-subscription)
-- [Next step: consume a data stream](/guides/guide-request-dataset/#next-step-consume-a-data-stream)
-
-## Prerequisites
-
-The only prerequisite for sending a Request is that a dataset from another member/company must be published to your DataSpace Ecosystem.
+# Subscribe to Dataset
+
+After the setup of the consumer node is complete, it can subscibe to the dataset that was created before.
```

Comment:
Not your changes but could we fix the sentence in line 28:

"If the owner of the dataset has defined terms and conditions of use, you have to agree on before you can send a Request to the owner." 

you have to agree with those before 

(those missing)?

Reply:
Also, the links in the the last paragraph of this file are broken due to the restructuring changes